tests/data/drivers/test_azure_kusto_driver.py

```diff
@@ -204,17 +204,16 @@ class ConnectTest(NamedTuple):
         init_args={"connection_str": "https://test1.kusto.windows.net"},
         connect_args={"connection_str": "https://test.kusto.windows.net"},
         tests=[
-            lambda driver: driver.client._kusto_cluster
-            == "https://test.kusto.windows.net"
+            lambda driver: driver.client._kusto_cluster.startswith(
```

Comment:
Is this meant to be included in this PR? I thought at first that it was just the code formatter, but it looks semantically different from the previous version. 

[Edit: "this" meaning the kusto testing related changes]

Reply:
Yep - definite typo

Reply:
Actually, it isn't a typo. This test started to break on this PR because of a change in the Kusto SDK. This test is checking the value for the URL in the SDK client. This used to be just the cluster URL but became URL+some path or query params.
However, I did find a bug related to this so thanks!!